tools/bridge/bridge/config.py

```diff
@@ -121,6 +121,13 @@ def validate_private_key(private_key: Any) -> dict:
     return private_key
 
 
+def validate_webservice(webservice_dict: Dict) -> Dict:
+    """validate the webservice dictionary"""
+    if not isinstance(webservice_dict, dict):
+        raise ValueError("webservice must be a dictionary")
+    return webservice_dict
```

Comment:
We should also check here that if it's enabled a host and port is given and that there are no additional entries.

Reply:
yes, this is a shortcut I took. Contrary to what I said in the daily I'm not sure what is the best way to solve this. Maybe we should just manually check that here.